src/media/css/header.styl

```diff
@@ -16,7 +16,12 @@ mkt-header-nav {
     order: 3;
 
     +media--wide-tablet() {
-        margin-right: 60px;
+        +ltr() {
+            margin-right: 60px;
```

Comment:
I'd rather these things have even margins than special case them for RTL/LTR. The header has 40px of padding but we can give it 35px by removing `padding: 0 40px` (it then falls back to `padding: 0 5px`) from it and setting `margin: 0 30px` on the search toggle, settings toggle and logo. This works fine though.

![screenshot 2015-06-11 09 52 11](https://cloud.githubusercontent.com/assets/211578/8110014/8f2a1ae8-101f-11e5-9e2e-e352d7158ef3.png)
> Current

![screenshot 2015-06-11 09 49 11](https://cloud.githubusercontent.com/assets/211578/8109996/7332d596-101f-11e5-852a-2c6445059a92.png)
> Updated mock up on dev